pynndescent/pynndescent_.py

```diff
@@ -32,7 +33,7 @@
 
 
 def make_initialisations(dist, dist_args):
-    @numba.njit(parallel=True, fastmath=True)
```

Comment:
I haven't benchmarked the effect of `fastmath=True`, but I think it's harmless to keep it.